packages/account/build/constants.js

```diff
@@ -17,6 +17,7 @@ const {
 } = require('./loaders-config');
 
 const ALIASES = {
+    'react/jsx-runtime': 'react/jsx-runtime.js',
```

Comment:
What is this for?

Reply:
we need this because we are trying to install `deriv-com/quill-ui` which they are using React v18 while our React version is v17. It causes the issue where it couldn't read `react/jsx-runtime`. This will solve the issue by replacing `react/jsx-runtime` with `react/jsx-runtime.js`